### PR TITLE
menu rejects invalid answers, has a 'not available yet' error

### LINKED ISSUE #58: Should we have a "not an option" error
### Context
You can enter `358` or `8` and it will do the same thing. Separate error message for "not an option"?

### PR DESCRIPTION
**Pull request by kenziebottoms**
# Description
Menu now has meaningful error messages instead of exiting without explanation.

## Related Ticket(s)
fixes #58 

## Problem to Solve
The exit option, invalid options, and under construction options all just exit without explanation.

## Expected Behavior
- Selecting `3`-`7` should give you "Please select an active customer."
- Selecting `8`-`11` should give you "This option is not available at this time."
- Selecting anything above 12 or below 1 should give you "RTFM"
- `12` is the only option that should kick you all the way out. 

## Steps to Test Solution
1. Try to break the above

## Testing

- [ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
- [x] I certify that all existing tests pass

## Documentation
- [ ] There is new documentation in this pull request that must be reviewed..
- [ ] I added documentation for any new classes/methods

### CODE REVIEW

File: app/ui.js [hunk 1]
```diff
@@ -45,13 +48,15 @@ let mainMenuHandler = (err, userInput) => {
...
-    } else if (userInput.choice == '3') {
+    } else if (intChoice == 3) {
       newPaymentOption(getActiveCustomer())
         .then(paymentObj => {
           return saveNewPaymentOption(paymentObj);
         })
         .then((response) => success(response))
         .catch(err => warning(err));
+    } else if (intChoice != 12) {
+      warning("This option is not avaible at this time.");
```
Comment (DavidLarsKetch) [inline app/ui.js]: This should move up to around line 41 and have a condition like `intChoice > 7`

**Review by DavidLarsKetch — DISMISSED** [review]
:+1:

**Review by jordan-castelloe — APPROVED** [review]
👍

**Review by joechesney — APPROVED** [review]
sweet

**Pull request merged**